dev/Scroller/APITests/ScrollerLayoutTests.cs

```diff
@@ -244,6 +244,157 @@ public void BasicAlignment()
             Verify.AreEqual(zoomFactor, c_smallZoomFactor);
         }
 
+        [TestMethod]
+        [TestProperty("Description", "Validates Scroller.ViewportHeight for various layouts.")]
+        public void ViewportHeight()
+        {
+            for (double scrollerContentHeight = 50.0; scrollerContentHeight <= 350.0; scrollerContentHeight += 300.0)
```

Comment:
So just 50 and 350, right?
And each time, 9 configurations. So 18 runs of the test in total?

Reply:
Yes indeed. It's real fast too.